db/models.py

```diff
@@ -45,3 +48,61 @@ class MovieSession(models.Model):
 
     def __str__(self):
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class User(AbstractUser):
+    pass
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```

Comment:
It's better to use variable AUTH_USER_MODEL from settings here instead of just "User" (best practice)

Reply:
Thanks, fixed